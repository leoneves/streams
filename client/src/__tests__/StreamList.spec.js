import React from 'react'
import { shallow } from 'enzyme'

import StreamList from '../components/streams/StreamList'

describe(StreamList, () => {
  context('with empty streams', () => {
    it('should return empty stream list',() => {
      debugger
      const wrapper = shallow(<StreamList streams={[]} />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
