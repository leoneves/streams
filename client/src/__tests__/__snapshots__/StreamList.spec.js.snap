// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect with empty streams should return empty stream list 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
